boards/ssrc/icicle/nuttx-config/scripts/script.ld

```diff
@@ -20,30 +20,21 @@
 
 MEMORY
 {
-  progmem (rx) : ORIGIN = 0xAFB00000, LENGTH = 3072K  /* w/ cache */
```

Comment:
Maybe we need to reserve 1 full page for this. Or figure out another solution, the ToC does not need to be "flashed" I think. So we could simply append the ToC, read it to a temporary buffer in the bootloader, and then "flash" the firmware only.

Reply:
Nvm, the ToC has to be flashed, otherwise the bootloader does not know where the app payload starts after re-boot.